benches/deadlock_detector/mod.rs

```diff
@@ -4,20 +4,21 @@ use criterion::{Bencher, Criterion};
 use kvproto::deadlock::*;
 use rand::prelude::*;
 use tikv::server::lock_manager::deadlock::DetectTable;
+use tikv::storage::TimeStamp;
 use tikv_util::time::Duration;
 
 struct DetectGenerator {
     rng: ThreadRng,
-    range: u64,
-    timestamp: u64,
+    range: TimeStamp,
```

Comment:
ISTM in this bench, `range` and `timestamp` don't look like a real `Timestamp`, especially `range`. Is it better to keep it as `u64`? @youjiali1995

Reply:
Oh, yes. `range` is `u64`.

Reply:
:+1: